uaclient/security.py

```diff
@@ -824,8 +836,15 @@ def _handle_released_package_fixes(
 
             if not upgrade_status:
                 unfixed_pkgs += [src_pkg for src_pkg, _ in pkg_src_group]
+            else:
+                installed_pkgs += binary_pkgs
```

Comment:
Can this new line go in the if statement above?
```suggestion
                installed_pkgs += binary_pkgs
                
            if not upgrade_status:
                unfixed_pkgs += [src_pkg for src_pkg, _ in pkg_src_group]
```

Reply:
The preceding if statement is `if upgrade_status:`

So I'm suggesting this new line go in that preceding if statement. Then we could actually turn this `if not upgrade_status` into an `else:`
```python
if upgrade_status:
    # all the stuff above
    # ....
    installed_pkgs.update(binary_pkgs)
else:
    unfixed_pkgs.update([src_pkg for src_pkg, _ in pkg_src_group])
```